src/Illuminate/Mail/Mailer.php

```diff
@@ -383,7 +383,7 @@ protected function parseView($view)
      * Send a Swift Message instance.
      *
      * @param  \Swift_Message  $message
-     * @return void
+     * @return int|null
```

Comment:
What about `Mailer::send()`, which is said to return `mixed` while returns `$this->sendSwiftMessage($message)`?